OJP/OJP_Common.xsd

```diff
@@ -398,4 +398,41 @@
 			<xs:group ref="WebLinkGroup" minOccurs="0"/>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:annotation>
+		<xs:documentation>========================================== Sustainability ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:group name="SustainabilityGroup">
+		<xs:annotation>
+			<xs:documentation>Elements about sustainability. Primarily for TRIPs and LEGs.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="EmissionCO2" minOccurs="0">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="GramPerPersonKm">
+							<xs:annotation>
+								<xs:documentation>Gram by person and by kilometer of CO2 emmisions.</xs:documentation>
```

Comment:
SI would dictate kilogram.

Reply:
True, but grap per personkm is the standard for CO2. So I would leave it. Can you agree @skinkie

Reply:
http://www.aef.org.uk/downloads/Grams_CO2_transportmodesUK.pdf. So we do kilograms. 
Decision: kilogram